catroid/src/main/res/values/strings.xml

```diff
@@ -2069,6 +2069,6 @@ needs read and write access to it. You can always change permissions through you
     <string name="hertz_symbol">㎐</string>
 
     <string name="sans_serif_font" translatable="false">sans_serif</string>
-    <string name="serif_font" translatable="false">serif</string>
+    <string name="serif_font" translatable="false">serif</string><string name="edit">Edit</string><string name="data_value">Value</string>
```

Comment:
```suggestion
    <string name="serif_font" translatable="false">serif</string>
    <string name="edit">Edit</string>
    <string name="data_value">Value</string>
```

This should fix most of the lint warnings of https://jenkins.catrob.at/job/Catroid/job/PR-3925/1/analysis/

Reply:
Working on the changes